utils/mapper/mapper.go

```diff
@@ -0,0 +1,40 @@
+package mapper
+
+import (
+	"errors"
+	"soarca/models/cacao"
+)
+
+func Variables(variables cacao.Variables,
```

Comment:
Just for clarity, readability, and debugging, I would rather call these arguments something like "variablesInScope", "outArgs", "returnedVariables", "capabilityReturnArgs"